src/async_client.rs

```diff
@@ -84,27 +83,77 @@ impl AsyncClient {
     /// This fails when the node does not exist or the attribute cannot be read.
     ///
     /// [`read_value()`]: Self::read_value
+    #[allow(clippy::missing_panics_doc)]
```

Comment:
I was wondering if this method is still needed. But the consistency checks are probably worth keeping it in the public API if this is a common use case. And we need it for `read_value()` internally.